doc/source/theory/N3LO_ad.rst

```diff
@@ -362,13 +363,13 @@ and following :cite:`Falcioni:2023luc` we use:
         :align: center
 
         *   - :math:`f_1(N)`
-            - :math:`\frac{1}{(N-1)^2}`
+            - :math:`\frac{1}{(N-1)^2} + \frac{1}{N^2}`
```

Comment:
`qq` is unchanged here, right? so this is a typo?

Reply:
this was a typo indeed